l10n_it_sdi_channel/models/sdi.py

```diff
@@ -165,7 +165,7 @@ def receive_fe(
         for file_name, file_content in file_name_content_dict.items():
             attachments_values = self._process_single_fe(
                 file_name,
-                file_content,
+                base64.decodebytes(file_content),
```

Comment:
Concordo con https://github.com/OCA/l10n-italy/pull/2919#pullrequestreview-1087666584: questo metodo dovrebbe riceve i file pronti per essere processati, non qualcosa di già codificato.

> file_name_content_dict must be base64-encoded, as described in receive_fe documentation.

Allora ci sarebbe da correggere la sua documentazione, l'ho aggiunto in https://github.com/OCA/l10n-italy/pull/2916.